doc/using_pytorch.rst

```diff
@@ -566,94 +566,120 @@ The function should return a byte array of data serialized to content_type.
 
-Working with Existing Model Data and Training Jobs
-==================================================
 
-Attach to existing training jobs
---------------------------------
+Bring your own model
+====================
 
-You can attach a PyTorch Estimator to an existing training job using the
-``attach`` method.
+You can deploy a PyTorch model that you trained outside of SageMaker by using the ``PyTorchModel`` class.
+Typically, you save a PyTorch model as a file with extension ``.pt`` or ``.pth``.
+To do this, you need to:
+
+* Write an inference script.
+* Package the model artifacts into a ``tar.gz`` file.
+* Upload the ``tar.gz`` file to an S3 bucket.
+* Create the ``PyTorchModel`` object.
+
+Write an inference script
+-------------------------
+
+You must create an inference script that implements (at least) the ``model_fn`` function that calls the loaded model to get a prediction.
+
+**Note**: If you use elastic inference with PyTorch, you can use the default ``model_fn`` implementation provided in the serving container.
+
+Optionally, you can also implement ``input_fn`` and ``output_fn`` to process input and output.
+For information about how to write an inference script, see `Serve a PyTorch Model <#serve-a-pytorch-model>`_.
+Save the inference script as ``inference.py`` in the same folder where you saved your PyTorch model.
+
+Package model artifacts into a tar.gz file
+------------------------------------------
+
+The directory structure where you saved your PyTorch model should look something like the following:
+
+**Note:** This directory struture is for PyTorch versions 1.2 and higher. For the directory structure for versions 1.1 and lower, 
+see `For versions 1.1 and lower <#for-versions-1.1-and-lower>`_.
+
+::
+
+    |   my_model
+    |           |--model.pth
+    |
+    |           code
+    |               |--inference.py
+    |               |--requirements.txt
+
+Where ``requirments.txt`` is an optional file that specifies dependencies on third-party libraries.
+
+With this file structure, run the following command to package your model as a ``tar.gz`` file:
+
+``tar -czf model.tar.gz my_model code``
+
+Upload model.tar.gz to S3
+-------------------------
+
+After you package your model into a ``tar.gz`` file, upload it to an S3 bucket by running the following Python code:
 
 .. code:: python
 
-    my_training_job_name = 'MyAwesomePyTorchTrainingJob'
-    pytorch_estimator = PyTorch.attach(my_training_job_name)
+    import boto3
+    import sagemaker
+    s3 = boto3.client('s3')
 
-After attaching, if the training job has finished with job status "Completed", it can be
-``deploy``\ ed to create a SageMaker Endpoint and return a
-``Predictor``. If the training job is in progress,
-attach will block and display log messages from the training job, until the training job completes.
+    from sagemaker import get_execution_role
+    role = get_execution_role()
```

Comment:
is `role` used anywhere?

Reply:
Moved this to the next section where `role` is passed to the model constructor.